canonical/global-nav
# Ua to pro

## Done
Update the product info for UA to Pro
Bumped the version

## QA
- Pull the branch
- Run `dotrun serve`
- Visit the demo and check the Pro content is right under the Product dropdown

Fixes https://github.com/canonical/ubuntu.com/issues/12404
